app/forms/schools/add_participant_form.rb

```diff
@@ -98,7 +80,7 @@ def type=(value)
     end
 
     def school_cohort
-      @school_cohort ||= SchoolCohort.find(school_cohort_id)
+      @school_cohort ||= SchoolCohort.find_by(id: school_cohort_id)
```

Comment:
Just curiosity now, why this change?

Reply:
I've made the change which allows you to call `form.valid?` to execute all the steps validation. `find` was raising exception on validating a new form.